plugins/services/src/js/service-configuration/ServiceStorageConfigSection.js

```diff
@@ -31,160 +26,120 @@ class ServiceStorageConfigSection extends ServiceConfigBaseSectionDisplay {
-              },
-              {
-                heading: getColumnHeadingFn("Mode"),
-                prop: "mode",
-                className: getColumnClassNameFn(),
-                sortable: true
-              },
-              {
-                heading: getColumnHeadingFn("Container Mount Path"),
-                prop: "containerPath",
-                className: getColumnClassNameFn(),
-                sortable: true
-              }
-            ];
-
-            let shouldDisplayHostPath = false;
-
-            const volumesData = volumes.map(appVolume => {
-              // We don't want to mutate the appVolume value.
-              const volume = {
-                name: null,
-                size: null,
-                type: [],
-                isHostVolume: false
-              };
-
-              volume.containerPath = appVolume.containerPath;
-              volume.mode = appVolume.mode;
-
-              if (appVolume.persistent != null) {
-                volume.size = appVolume.persistent.size;
-                volume.type.push("Persistent", "Local");
-              } else if (appVolume.external != null) {
-                volume.size = appVolume.external.size;
-                volume.type.push("External");
-              } else {
-                volume.isHostVolume = true;
-                volume.type.push("Host", "Volume");
-              }
-
-              if (appVolume.external != null) {
-                volume.name = appVolume.external.name;
-              }
-
-              if (appVolume.hostPath != null) {
-                // Set this flag to true so that we render the hostPath column.
-                shouldDisplayHostPath = true;
-              }
-
-              volume.hostPath = getDisplayValue(
-                appVolume.hostPath,
-                !volume.isHostVolume
-              );
-
-              return volume;
-            });
+        }
+      ]
+    };
 
-            if (shouldDisplayHostPath) {
-              columns.push({
-                heading: getColumnHeadingFn("Host Path"),
-                prop: "hostPath",
-                className: getColumnClassNameFn(),
-                sortable: true
-              });
+    const volumesConfig = volumes.map((volume, index) => {
+      if (volume.persistent != null && volume.persistent.profileName != null) {
+        // DSS
+        return [
+          {
+            heading: VolumeDefinitions.DSS.name,
+            headingLevel: 2
+          },
+          {
+            key: `container.volumes.${index}.persistent.profileName`,
+            label: "PROFILE NAME"
+          },
+          {
+            key: `container.volumes.${index}.containerPath`,
+            label: "CONTAINER PATH"
+          },
+          {
+            key: `container.volumes.${index}.persistent.size`,
+            label: "SIZE",
+            transformValue: this.getVolumeSizeValue
+          }
+        ];
+      } else if (volume.persistent != null) {
```

Comment:
Personal opinion. Since we return and this exists, we could maybe remove the ``` } else if { ``` and just keep the if statements and just return the default one at the end.

Reply:
True, I have updated to only use `if`. thanks